src/mouse.js

```diff
@@ -4,7 +4,8 @@ var pointermap = dispatcher.pointermap;
 // radius around touchend that swallows mouse events
 var DEDUP_DIST = 25;
 
-var WHICH_TO_BUTTONS = [0, 1, 4, 2];
+// left, middle, right, back, forward
+var BUTTON_TO_BUTTONS = [1, 4, 2, 8, 16];
```

Comment:
> Are you aware of any overview on hardware/UA-combinations that send more button-codes?

I don’t have one to test but I know there are 10-button mice on the market. Hopefully this becomes a non-issue soon since I believe Chrome is the only one that [hasn’t implemented yet](https://code.google.com/p/chromium/issues/detail?id=276941).